metrotranscom/doorway
# feat: add mtc logo to the top nav, remove from footer

This PR addresses #934 


## Description

This adds the MTC logo to the left side (or above on mobile) of the languages nav bar.

## How Can This Be Tested/Reviewed?

Visit the Public site and verify the logo is available at the top-left and goes to the right website when clicked, and that the logo has been removed from the footer.

**Note:** in the Figma mockups linked in the issue, the language bar looks different than what's currently in Doorway production. If the code needs to be revised to match Figma's appearance, I'd recommend a follow-up ticket to address that.

**Note 2:** I did "Seeds-ify" the CSS of the language bar, and I would have also moved it out of doorway-ui-components except it's referenced by the SiteHeader component. Happy to discuss options.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
